contracts/system_addresses.go

```diff
@@ -28,4 +28,7 @@ var (
 	BLSAggSigsVerificationPrecompile = types.StringToAddress("0x2030")
 	// ConsolePrecompile is and address of Hardhat console precompile
 	ConsolePrecompile = types.StringToAddress("0x000000000000000000636F6e736F6c652e6c6f67")
+
+	// AccountAbstractionInvokerContract is address of account abstraction invoker contracts
+	AccountAbstractionInvokerContract = types.StringToAddress("0x3001")
```

Comment:
As of now, the invoker is not deployed during genesis as a special account. Anyone could deploy their own invoker.

Reply:
I would assume this is a parameter for the relayer and an external workflow deploys the contract.

Reply:
So we will not have default invoker? We than probably need invokerAddress as a parameter on both aa commands

Reply:
Removed. Because I did re-base,  change is in first commit.